doc/connectivity/parcellating.rst

```diff
@@ -144,3 +144,37 @@ representation thanks to a two-step procedure :
 We can see that using only 2000 parcels, the original image is well
 approximated.
 
+Another approach: spectral clustering
+=====================================
+
+Spectral clustering is a segmentation approach known for its simplicity and its
+(relatively) low computational cost. Spectral clustering uses the spectrum of an
```

Comment:
the computational cost is actually high !
It is well suited to cases where the data are characterized by their similarity (i.e. locally) rather than some global coordinates. To be pedantic, this is good at clustering data sampled on manifolds.